docs/openapi/open-api.yml

```diff
@@ -56,6 +78,53 @@ paths:
                     mTeam: './mteam-league.yml#/components/schemas/MTeamLeagueResponse'
                     mRoster: './mroster-league.yml#/components/schemas/MRosterLeagueResponse'
 
+  /{sportId}/seasons/{seasonYear}/segments/0/leagues/{leagueId}/transactions:
+    post:
+      summary: 'Submit lineup changes'
+      description:
+        POST request to submit lineup changes
+      parameters:
+        - name: sportId
+          in: path
+          description: >
+            The string based id of the fantasy sport.
+            * nfl - ffl
+            * nba - fba
+            * nfl - fhl
+            * mlb - mlb
+          required: true
+          schema:
+            type: string
+            enum:
+              - ffl
+              - fba
+              - fhl
+              - flb
+        - name: seasonYear
+          in: path
+          description: The year of the fantasy season (>2018).
+          required: true
+          schema:
+            type: integer
+        - name: leagueId
+          in: path
+          description: The id of the fantasy league.
+          required: true
+          schema:
+            type: integer
+      requestBody:
+        description: payload
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TransactionPayload'
+      responses:
+        '200':
```

Comment:
Is there actually a 200 response or is it just a 202?

Reply:
200

Reply:
What does the 200 response mean vs the 202?